state/application_test.go

```diff
@@ -1696,8 +1705,7 @@ func (s *ApplicationSuite) TestForceDoesNotAllowRemovingAnApplicationWithOffers(
 	op := s.mysql.DestroyOperation()
 	op.Force = true
 	err := s.State.ApplyOperation(op)
-	c.Assert(err, gc.ErrorMatches, `cannot destroy application "mysql": application is used by 1 offer`)
-	assertOffersRef(c, s.State, "mysql", 1)
+	c.Assert(err, jc.ErrorIsNil)
```

Comment:
we still need to check that things are removed, right?
the app itself and any offers